play/fraction-bingo/compare/app.js

```diff
@@ -0,0 +1,253 @@
+}
+
+function getRandomFraction(min, max) {
+    let num;
+    let numerator = Math.floor(Math.random() * (max - min + 1)) + min;
+    if (min == 0) {
+	min = 1;
+    }
+    let denominator = Math.floor(Math.random() * (max - min + 1)) + min;
+    //num = `${numerator}&frasl;${denominator}`;
+    num = numerator + "/" + denominator;
+    return num;
+}
+
+// Randomly fill board with <, >, and = 
+function fillBoard() {
+    table = document.getElementsByName(boardType)[0];
+    let operators = ["<", ">", "="];
+    let op = "";
+    for (var i = 0, row; row = table.rows[i]; i++) {
+	for (var j = 0, col; col = row.cells[j]; j++) {
+	    op = operators[Math.floor(Math.random() * 3)];                                 
+            col.className = "white";
+	    col.innerText = op;
+            col.onclick = function () {
+		checkAnswer(this);
+            };
+	}
+    }
+}
+
+function checkAnswer(cell) {
+    if (cell.className != "green") {
+        if (correctOp == cell.innerText) {
+	    score += 10;
+	    numSolved++;
+	    cell.className = "green";
+	    let b = checkBingo();
+	    if (!b) {
+		setTimeout(function() {
+			generateProblem();
+		}, 1000);
+	    }
+	}
+        else {
+	    if (score != 0) {
+		score -= 10;
+	    }
+	    $('.list-group').append("<li class='list-group-item'><h5 class='list-group-item-heading'>" + problem + "</h5> <p class='list-group-item-text'><b>Your Answer: </b>" + cell.innerText + "<br> <b\
+>Correct: </b>" + correctOp + "</p> </li>");
+       	    cell.className = "red";
+	    setTimeout(function() {
+	        cell.className = "white";
+	    }, 1000);
+        }
+	document.getElementById("score").innerHTML = "Score: " + score;
+    }
+}
+
+function solNotFound() {
+    noSolBtn = document.getElementById("noSolution");
+    for (var i = 0, row; row = table.rows[i]; i++) {
+        for (var j = 0, col; col = row.cells[j]; j++) {
+	    if (col.className == "white") { 
+	        if (correctOp == col.innerText) {
+		    if (score != 0) {
+			score -= 10;
+		    }
+		    document.getElementById("score").innerHTML = "Score: " + score;
+		    $('.list-group').append("<li class='list-group-item'><h5 class='list-group-item-heading'>" + problem + "</h5> <p class='list-group-item-text'><b>Your Answer: </b> Solution not on boar\
+d <br> <b>Correct: </b>" + correctOp + "</p> </li>");
+                    noSolBtn.style.backgroundColor = "red";
+                    setTimeout(function() {
+                        noSolBtn.style.backgroundColor = "";
+                    }, 1000);
+                    return;
+                }
+	    }
+        }
+    }
+    score += 10;
+    numSolved++;
+    document.getElementById("score").innerHTML = "Score: " + score;
+    noSolBtn.style.backgroundColor = "green";
+    setTimeout(function() {
+        noSolBtn.style.backgroundColor = "";
+	generateProblem();
+    }, 1000);
+}
+
+function checkBingo() {
+    let result;
+    result = checkRows();
+    if (result) {
+        return;
```

Comment:
Do you want to return ```true``` or ```false``` during this process? One way you could do this would be
```js
return checkRows() || checkColumns() || checkDiagonals();
```